libraries/stdlib/genosl/mx_image_color3.osl

```diff
@@ -1,8 +1,8 @@
 #include "$fileTransformUv"
 
-void mx_image_color3(string file, string layer, color default_value, vector2 texcoord, string uaddressmode, string vaddressmode, string filtertype, string framerange, int frameoffset, string frameendaction, output color out)
+void mx_image_color3(FILETEXTURE file, string layer, color default_value, vector2 texcoord, string uaddressmode, string vaddressmode, string filtertype, string framerange, int frameoffset, string frameendaction, output color out)
```

Comment:
[Newbie comment] Not a big fan of the name as `file` & `texture` are partially redundant.

Reply:
[Newbie comment] What about not color managed textures?

Reply:
It's not always color managed (e.g. in the case of a vector input) so maybe not that. 
I guess I can use the TEXTURE as @niklasharrysson suggested but was worried about name clashing.